### PR TITLE
fix monero-miner.com

### PR DESCRIPTION
**Pull request by SMed79**

### CODE REVIEW

**Comment by SMed79** [comment]
@hoshsadiq 
To avoid the circumvent using the **FQDN**,[1] I want to update the `! TLD blocking` section by:
```
/^(https?|wss?):\/\/([0-9a-z\-]+\.)?([0-9a-z\-]+\.)(accountant|bid|cf|club|cricket|date|download|faith|fun|ga|gdn|gq|loan|men|ml|ovh|party|pro|pw|racing|review|rocks|science|site|space|stream|tk|top|trade|webcam|win|xyz|zone)(\.)?\/(.*)/$third-party,websocket,domain=~myinterview.com|~webtorrent.io
```
Do you approve?

[1] https://en.wikipedia.org/wiki/Fully_qualified_domain_name
 (e.g [#109902](https://forums.lanik.us/viewtopic.php?p=109902#p109902)) 12 Dec 2016

@ryanbr fyi.

**Comment by hoshsadiq** [comment]
Agreed, otherwise we'll have to duplicate every TLD. Looks fine to me. Do you want to add a commit to this or open a new PR for that?

File: hosts.txt [hunk 1]
```diff
@@ -155,6 +155,7 @@
 0.0.0.0 flophous.cf
 0.0.0.0 pr0gram.org
 0.0.0.0 authedmine.eu
+0.0.0.0 www.monero-miner.com
```
Comment (hoshsadiq) [inline hosts.txt]: better to have this without www

Comment (SMed79) [reply hosts.txt]: `monero-miner.com` redirect me to `www.monero-miner.com`

**Comment by SMed79** [comment]
> Do you want to add a commit ... 

I let you do the job.

**Comment by hoshsadiq** [comment]
https://github.com/hoshsadiq/adblock-nocoin-list/commit/e2c961fca4834e653ecb5faa4e7bd5c4ec4a4540

**Pull request merged**